coalib/output/ConsoleInteraction.py

```diff
@@ -199,6 +199,112 @@ def acquire_actions_and_apply(console_printer,
             break
 
 
+def get_context(console_printer, file_dict,
+                sourcerange):  # pragma: no cover
+    """
+    Gets context for the affected lines and also the line numbers
+    of the context
+
+    :param file_dict:       A dictionary containing all files as values with
+                            filenames as key.
+    :param sourcerange:     The SourceRange object referring to the related
+                            lines to print.
+
+    :yield:                 Affected file's contextual information
+
+    """
+
+    related_lines = list()
+    related_lines_nr = list()
+
+    affected_line = file_dict[sourcerange.file][
+                    sourcerange.start.line-1].rstrip()
+    affected_line = affected_line.replace('\t', 8*' ')
+
+    indet_space_count = len(affected_line)-len(affected_line.lstrip())
+    if indet_space_count > 0:
+        for j in range(sourcerange.start.line-2, -1, -1):  # pragma: no cover
+            rline = file_dict[sourcerange.file][j].rstrip()
+            affected_line = rline.replace('\t', 8*' ')
+            prv_space_count = len(affected_line) - len(affected_line.lstrip())
+            if prv_space_count < indet_space_count and rline:
```

Comment:
prv_space_count exists because of affected_line which in turn was derived from rline. So rline would be not None irrespective? Also please change name of variable for `prv_space_count`. Something which is more clear to read and understand.